fw/vhost.c

```diff
@@ -2729,60 +2965,77 @@ static TfwCfgSpec tfw_global_frang_specs[] = {
 		},
 		.allow_reconfig = true,
 	},
-	/* Option can be redefined per vhost|location. */
+	/* Option can be redefined per vhost|location.
+	 *
+	 *  All handler are changed to tfw_cfgop_frang_glob_...
+	 *  because here we need to compare the default value
+	 *  with 0 after kzalloc, and for non global configuration
```

Comment:
I don't think that it is ok. For `http_body_len` we have different behavior then for `http_uri_len` for example.